StatPlugin.cs

```diff
@@ -4,47 +4,47 @@
 
 namespace DNWS
 {
-  class StatPlugin : IPlugin
```

Comment:
Don’t mess up with tab size, otherwise, you will have lots of unnecessary changes.